### ISSUE TITLE
Refresh Token Doesn't gone

### ISSUE THREAD

**Issue by Sagar-Puniyani** [hiteshchoudhary/chai-backend#27]
When the **`refreshToken`** is generated, while login the user then, it will stored in the database 
then the next step is to logout the user.

Logout of the user there is a problem starts 

while logout **`refreshToken`**  should dropped from the database.
as per the logic, the refresh token should be deleted from the database.

step 1: Register user 
![image](https://github.com/hiteshchoudhary/chai-backend/assets/126869215/03b6af48-561e-42e8-a685-ea3c02b75d6d)

`User Data in DataBase`
![image](https://github.com/hiteshchoudhary/chai-backend/assets/126869215/c87caa19-d4f4-4ca3-a54d-26b5b4302fb2)

step 2: Login the User 
![image](https://github.com/hiteshchoudhary/chai-backend/assets/126869215/1223fd91-cbe0-429c-a5f7-7fda3d2a31d8)

`RefreshToken is there in the Database`
![image](https://github.com/hiteshchoudhary/chai-backend/assets/126869215/109848cf-645f-4f6d-a68d-db5888c21cc8)

step 3:  Logout the User 
![image](https://github.com/hiteshchoudhary/chai-backend/assets/126869215/c5c8d489-1a40-4046-b2a8-93c6c6601f04)

**`cookies are gone but from the database, the refresh token is still there`**
![image](https://github.com/hiteshchoudhary/chai-backend/assets/126869215/1bbfc52e-c864-45db-816a-087f145b3bc1)

**Comment by Sagar-Puniyani** [comment]
It is the fix of #28

**Comment by anishudupa** [comment]
I guess it's your wish. Personlly it set it into empty string when user logs out.

**Comment by Sagar-Puniyani** [comment]
@anishudupa Yes, you are not wrong but in any case, you have to remove the previous value of  `refreshToken`

**Comment by anishudupa** [comment]
@Sagar-Puniyani yea true. It's all developer's wish tbh 🤣
